packages/nimble-tokens/CONTRIBUTING.md

```diff
@@ -61,13 +64,17 @@ For any token metadata changes (e.g. documentation, code snippets, etc.):
    4. Choose to replace any existing files in the `dist/icons/svg` folder.
 
       <img src="docs/ai-export-4.png" width="600"> 
+3. Proceed to the steps below or [create a user story](https://github.com/ni/nimble/issues/new/choose) requesting that the Nimble team perform them. If filing an issue, attach the new and modified SVG files and also `Nimble_Iconography.ai` to the issue.
+
+### Adding icons to Nimble
 
-4. Search for all `<defs>.*</defs>` tags in the exported `.svg` files (in the `packages/nimble-tokens/dist/icons/svg` directory), and remove them. This removes all color from the `.svg` files and allows us to dynamically change the fill color.
+1. In a new branch, copy the Illustrator file to `source/icons/Nimble_Iconography.ai` and the SVG files to `dist/icons/svg`.
+2. Search for all `<defs>.*</defs>` tags in the exported `.svg` files and remove them. This removes all color from the `.svg` files and allows us to dynamically change the fill color.
 
       <img src="docs/find-replace-5.png" width="1000">
 
-5. Confirm the new icon files will build correctly by running: `npm run build -w @ni/nimble-tokens` & `npm run build -w @ni/nimble-components`.
-6. Add metadata for the new icons to `nimble-components\src\icon-base\icon-metadata.ts`.
-7. Generate and build icon components by running `npm run build -w @ni/nimble-components`.
-8. Preview the built files by running: `npm run storybook -w @ni/nimble-components`, and review the **Icons** story to confirm that your changes appear correctly.
-9. Publish a PR with your changes. If there are any new icons, set `changeType` and `dependentChangeType` to minor in the beachball change file.
+3. Confirm the new icon files will build correctly by running: `npm run build -w @ni/nimble-tokens`.
```

Comment:
Good point. I had moved the step because it had failed when I ran it but I didn't register that it had necessary side effects.

To address this I added steps instructing you to build `nimble-components` twice and explained that the first one will fail.